kratos/tests/cpp_tests/containers/test_flags.cpp

```diff
@@ -120,16 +120,16 @@ KRATOS_TEST_CASE_IN_SUITE(KratosFlagsSetMultiple, KratosCoreFastSuite) {
 
 KRATOS_TEST_CASE_IN_SUITE(KratosFlagsEquality, KratosCoreFastSuite) {
 
-    KRATOS_CHECK_EQUAL( INLET == NOT_INLET, false );
-    KRATOS_CHECK_EQUAL( INLET != NOT_INLET, true );
+    KRATOS_CHECK_EQUAL( INLET == Flags::Unset(INLET), false );
+    KRATOS_CHECK_EQUAL( INLET != Flags::Unset(INLET), true );
     // trivial check (it is acutally 0 == 0) just to see that everything works when the first argument of == is not a lhs
-    KRATOS_CHECK_EQUAL( (~INLET & INLET) == NOT_INLET, true );
+    KRATOS_CHECK_EQUAL( (Flags::Unset(INLET) & INLET) == Flags::Unset(INLET), true );
```

Comment:
I think ~ is the correct operator instead of Unset... Isn't it?

Reply:
Using the `~` doesn't yield the same result (it's the first thing I tested).